GameOfLifeApp/View/Display.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Displays 2 boolean values arrays. 
+    /// </summary>
+    public class Display : IDisplay
+    {
+        private IGameField pr;
```

Comment:
why do you have it as private here?

Reply:
Removed. It wasn`t necessary. I passed interface as a parameter into the method.